### PR TITLE
Rename variable wait to interval

### PR DESCRIPTION
**Pull request by AndersonChoi**
The interval seems to be a better word than wait.

### CODE REVIEW

**Comment by AndersonChoi** [comment]
@thockin thanks for the feedback - I made your changes so this is ready for another look :)

File: cmd/git-sync/main.go [hunk 1]
```diff
@@ -50,8 +50,11 @@ var flRoot = flag.String("root", envString("GIT_SYNC_ROOT", envString("HOME", ""
 	"the root directory for git operations")
 var flDest = flag.String("dest", envString("GIT_SYNC_DEST", ""),
 	"the name at which to publish the checked-out files under --root (defaults to leaf dir of --repo)")
+// Deprecated: flWait is deprecated. Use flInterval instead.
```
Comment (thockin) [inline cmd/git-sync/main.go]: Can you also add "(deprecated, see --interval)" to the string description?

File: cmd/git-sync/main.go [hunk 2]
```diff
@@ -144,6 +147,16 @@ func main() {
 		flag.Usage()
 		os.Exit(1)
 	}
+	if *flOneTime == false && *flWait != 0 {
+		if *flInterval == 0 {
+			log.Errorf("wait is deprecated. use interval instead")
```
Comment (thockin) [inline cmd/git-sync/main.go]: log.V(0).Infof please

Comment (thockin) [inline cmd/git-sync/main.go]: "The --wait flag is deprecated, please use --interval instead."

File: cmd/git-sync/main.go [hunk 3]
```diff
@@ -144,6 +147,16 @@ func main() {
...
+			*flInterval = *flWait
+		} else {
+			fmt.Fprintf(os.Stderr, "ERROR: use only one --interval or --wait. These cannot be used together.\n")
```
Comment (thockin) [inline cmd/git-sync/main.go]: "ERROR: flags --wait and --interval are mutually exclusive" is sufficient

File: cmd/git-sync/main.go [hunk 4]
```diff
@@ -144,6 +147,16 @@ func main() {
 		flag.Usage()
 		os.Exit(1)
 	}
+	if *flOneTime == false && *flWait != 0 {
```
Comment (thockin) [inline cmd/git-sync/main.go]: Can you add a separate check - if --wait and --one-time are specified, it should be an error.  We can't make it an error (breaking change) but we can make it log.

File: cmd/git-sync/main.go (hunk 3 above)
Comment (AndersonChoi) [reply cmd/git-sync/main.go]: I got it.

File: cmd/git-sync/main.go (hunk 4 above)
Comment (AndersonChoi) [reply cmd/git-sync/main.go]: Thank you for comment. I added it.

**Comment by AndersonChoi** [comment]
@thockin Thank you for your feedback. The details have been applied.

**Comment by AndersonChoi** [comment]
Hello @thockin :) 
If you have any further changes to make, please let me know and I'll apply them. Thank you. Have a nice day.

File: test_e2e.sh [hunk 5]
```diff
@@ -285,7 +285,7 @@ assert_file_exists "$ROOT"/link/file
 assert_file_eq "$ROOT"/link/file "$TESTCASE 1"
 # Wrap up
 remove_sync_container
-wait
+interval
```
Comment (thockin) [inline test_e2e.sh]: These changes are wrong - `wait` here is a shell command.

Comment (AndersonChoi) [reply test_e2e.sh]: I apologize for my mistake. 
I reverted command.

**Comment by AndersonChoi** [comment]
Hi @thockin . I apologize for my mistake.
I reverted command.

**Comment by thockin** [comment]
sorry, it seems you have a conflict with another PR that went it.

**Comment by AndersonChoi** [comment]
I'm really sorry to resolve conflict source late. 
Would you check my pull request source please?
I greatly appreciate your review.

**Pull request closed without merging**